docs/getting-started/overview.rst

```diff
@@ -1,6 +1,45 @@
-.. include:: /stub-topic.txt
+Overview of different editions of .NET
+======================================
+
+With the advent of .NET Core and .NET Native, the layout of the .NET ecosystem has changed significantly. Where before there was a (somewhat) simple mapping between platforms and .NET Framework versions, today we have a lot of 
```

Comment:
Yes, I was thinking about putting in Mono as well, but opted out of it for speed to get this version out for comments.